src/main/js/portal/main/models/Preview.ts

```diff
@@ -60,13 +69,14 @@ export default class Preview {
 		this.pids = this.items.map(item => getLastSegmentInUrl(item.dobj));
 		this.options = options ?? [];
 		this.type = type;
-		this.previewSettings = this.item?.urlParams ? Preview.allowlistPreviewSettings(this.item.urlParams) : {};
+		this.previewSettings = Preview.allowlistPreviewSettings(this.item?.urlParams);
 	}
 
-	static allowlistPreviewSettings(urlParams: IdxSig | PreviewSettings): PreviewSettings {
-		const allowedPreviewSettings: PreviewSettings = {};
+	static allowlistPreviewSettings(urlParams?: IdxSig | PreviewSettings): PreviewSettings {
```

Comment:
This seems more informative, I think?

```suggestion
	static allowlistPreviewSettings(urlParams?: Partial<PreviewSettings>): PreviewSettings {
```

I also think we could rename this to something like `buildPreviewSettings` or similar?

Reply:
Agreed on both counts!